src/main/java/br/com/meli/exercises/controller/LinkController.java

```diff
@@ -0,0 +1,54 @@
+package br.com.meli.exercises.controller;
+
+import br.com.meli.exercises.dtos.LinkIdResponse;
+import br.com.meli.exercises.dtos.LinkUrlRequest;
+import br.com.meli.exercises.model.Link;
+import br.com.meli.exercises.repositories.LinksRepository;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+
+@RestController
+@RequestMapping("/link")
+public class LinkController {
+
+    private final LinksRepository linksRepository;
+    LinkController(LinksRepository linksRepository){
+        this.linksRepository = linksRepository;
+    }
+
+    @PostMapping()
```

Comment:
Não precisa abrir parênteses aqui se for deixa tudo como default.

```suggestion
    @PostMapping
```